### PR TITLE
Solstice Wreaths and trees

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
SUMMARY: Content "Holiday Wreaths and Trees"

#### Purpose of change
Adds seasonal holiday items for the winter solstice/Christmas.

#### Describe the solution
Recipes, items and furniture.

#### Describe alternatives you've considered
None

#### Testing
TBD

#### Additional context
None

### CODE REVIEW

**Comment by ymber** [comment]
Why isn't it called a Christmas tree? I've never heard anyone call it a solstice tree.

**Comment by Amoebka** [comment]
Should probably be called "politically neutral winter holiday tree" instead.

**Comment by Maleclypse** [comment]
Because all my friends who bother to put up trees are pagan and because whether Christmas survives post cataclysm with the resurrection trying to murder you the celestial event that Christmas commemorates will still exist. :) but if anyone is worried the PC police are infiltrating their safe space I can rename it to whatever.

**Comment by Fris0uman** [comment]
`Decorated Tree` might be a slightly clearer name

**Comment by Maleclypse** [comment]
> `Decorated Tree` might be a slightly clearer name, and since it's purely descriptif it's flavour neutral and players can decide if it's a pagan, a christian, a capitalist or a whatever thing.

Perfect :) done.

**Comment by Maleclypse** [comment]
Here is the fail I was able to find `Backtrace emission took 2 seconds.
src/json.cpp:112: Failed to visit member 'entries' in JsonObject at line 320:14277:
{
    "id": "nanomaterials",
    "type": "item_group",
    "entries": [ { "item": "nanomaterial", "prob": 100 } ]`

File: data/json/recipes/recipes_holiday.json [hunk 1]
```diff
@@ -15,5 +15,17 @@
...
+    "type": "recipe",
+    "result": "decorative_wreath",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "45 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "pine_bough", 4 ] ], [ [ "duct_tape", 10 ] ] ]
```
Comment (I-am-Erk) [inline data/json/recipes/recipes_holiday.json]: You could include cordage as an alternative to duct tape.

**Pull request merged**